test/unit/DetectChanges/BCBreak/TraitBased/PropertyRemovedTest.php

```diff
@@ -0,0 +1,84 @@
+<?php
```

Comment:
Can we drop this before merging? In theory, all changes are covered by the `ReflectionPropertyNameTest`

Reply:
Ok, I've removed the test!